.github/ISSUE_TEMPLATE/20-feature_request.yaml

```diff
@@ -0,0 +1,65 @@
+name: Feature
+description: |
+    Suggest a new feature for next Bareos
+    (**NOTE**: we only making critical & bug fixes to Bareos current)
+title: Bareos Next - Feature request
```

Comment:
```suggestion
title: "[FEATURE REQUEST]: "
```

Reply:
like for bug we will remove this and let it blank